spot_driver/CMakeLists.txt

```diff
@@ -85,7 +86,7 @@ target_include_directories(spot_api PUBLIC
   $<INSTALL_INTERFACE:include>
 )
 
-target_link_libraries(spot_api PUBLIC bosdyn::bosdyn_client_static)
+target_link_libraries(spot_api PUBLIC bosdyn::bosdyn_client)
```

Comment:
Is this same change also needed for the ros2 control package?

Reply:
Good catch. Done in https://github.com/bdaiinstitute/spot_ros2/pull/495/commits/93d3b304feac6f10fa26013c81111c4f6483acb5.